api/routes/items.ts

```diff
@@ -190,6 +191,37 @@ router.post("/archive", isUser, async (req: Request, res: Response) => {
   return res.status(200).send({ msg: updatedItems });
 });
 
+// Archive all items older than 90 days.
+async function archiveOldItems() {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const ObjectID = require("mongodb").ObjectID;
+  return Item.updateMany(
+    {
+      $and: [
+        {
+          archived: false,
+        },
+        {
+          _id: {
+            $lt: ObjectID.createFromTime(Date.now() / 1000 - 90 * 24 * 60 * 60),
+          },
+        },
+      ],
+    },
+    [
+      {
+        $set: {
+          archived: true,
+          dateArchived: new Date(),
+          archiver: `Automatically Archived on ${new Date().toDateString()}`,
+        },
+      },
+    ]
+  );
+}
+// Once a week, automatically archives all items older than 90 days
+schedule("0 0 * * *", archiveOldItems);
```

Comment:
I think this does every day at 12 am.

Do you mean `0 0 * * 0` to mean at 12 am on every Sunday?